caddyhttp/browse/browse_test.go

```diff
@@ -70,6 +70,13 @@ func TestSort(t *testing.T) {
 		t.Errorf("The listing isn't time sorted: %v", listing.Items)
 	}
 
+	// sort by name dir first
+	listing.Sort = "namedirfirst"
+	listing.applySort()
+	if !sort.IsSorted(byNameDirFirst(listing)) {
+		t.Errorf("The listing isn't namedirfirst sorted: %v", listing.Items)
+	}
+
```

Comment:
Those tests are not needed, IMHO. There's no formal *contract* as to what order is the default. Adding code = adding complexity.

But there's admittedly several families of thinking regarding what and how granular to test.